database/sql/src/main/scala/cromwell/database/slick/tables/WorkflowStoreEntryComponent.scala

```diff
@@ -25,8 +25,10 @@ trait WorkflowStoreEntryComponent {
 
     def submissionTime = column[Timestamp]("SUBMISSION_TIME")
 
+    def importsZipFile = column[Option[Blob]]("IMPORTS_ZIP_PATH")
```

Comment:
I believe currently it's printing the byte content based on some centaur testing, but that doesn't seem to be your intent? 

TOL: Do we want the path or the content? We don't return paths for the wdl, inputs, options. Maybe it makes sense to present the user with the byte content, the way you have it now.